language/compiler/ir_to_bytecode/syntax/src/ast.rs

```diff
@@ -143,13 +145,30 @@ pub enum StructDefinitionFields {
 #[derive(Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Clone)]
 pub struct FunctionName(String);
 
+/// The kind of a type
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum Kind {
+    All,
```

Comment:
Comments for each of these might be useful even though they are already documented in the file format.

Reply:
Will do.